darwin/Platform.c

```diff
@@ -126,7 +126,7 @@ static double Platform_nanosecondsPerMachTick = 1.0;
 
 static double Platform_nanosecondsPerSchedulerTick = -1;
 
-void Platform_init(void) {
+int Platform_init(void) {
```

Comment:
Why change this, if the only return value is `0` anyways? If this shares the same semantics as `parseArguments` then a similar (or the same) enum should be used for marking the return value.